config/owasp/suppressions.xml

```diff
@@ -91,4 +91,18 @@
     <packageUrl regex="true">^pkg:maven/io\.netty/netty.*@4\.1\.69\.Final$</packageUrl>
     <cve>CVE-2022-24823</cve>
   </suppress>
+  <suppress until="2022-07-01">
+    <notes><![CDATA[
+   file name: common-image-3.7.0.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com\.twelvemonkeys\.common/common\-.*$</packageUrl>
```

Comment:
Did you add 2 of the same suppressions?

Reply:
No, different regexs

Reply:
different suppressions but streamlined the regex to include both

Reply:
that's what I call being a boss @LTurkingtonHMCTS 💪